src/components/AddNewBook.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Book.css';
+
+function Book() {
+  return (
+    <>
+      <hr />
+      <h2>ADD NEW BOOK</h2>
+      <form action="">
+        <div className="formcontrol">
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add title"
+            name="title"
+          />
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add author"
+            name="author"
+          />
+          <button type="button" className="addBtn">Add Book</button>
+        </div>
+      </form>
+    </>
+  );
+}
```

Comment:
- [ ] You named this function `Book` making you have two books functions. Kindly rename this function and then delete the form from the previous component `Book.js` where you created it earlier and only call this function there so that the add button shows on your screen.